tasks/src/table-question-answering/about.md

```diff
@@ -0,0 +1,37 @@
+## Use Cases
+
+### SQL execution
+
+You can use Table Question Answering model to simulate SQL execution by using provided table.
```

Comment:
```suggestion
You can use the Table Question Answering models to simulate SQL execution by inputting a table.
```

Reply:
Can you elaborate this a bit more, what do you input and what do you get after the inference?